index.html

```diff
@@ -45,5 +45,13 @@ <h1 class="Header-title logo" data-logo="Metalsmith">
 
   <script type="text/javascript" src="/build.js"></script>
   <script type="text/javascript">require('metalsmith.io');</script>
+  <script src="https://cdn.jsdelivr.net/npm/blazy@1.8.2/blazy.min.js"></script>
+  <script>
```

Comment:
The can go in https://github.com/metalsmith/metalsmith.io/blob/master/index.js when you're happy.
Don't forget to reorder blazy :)

Reply:
Tried to do minimal changes, but yeah, can do it while fixing the filtering :)